libraries/Microsoft.Bot.Connector/Authentication/SimpleChannelProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Threading.Tasks;
+
```

Comment:
REmove & Sort usings (I don't think you're using LINQ anywhere)

Reply:
Eyes like an eagle, he has!